steps/start/postcode-checker/BranchForEnglandOrWales.js

```diff
@@ -0,0 +1,64 @@
+const { branch, goTo } = require('@hmcts/one-per-page');
+const request = require('request');
+const config = require('config');
+const { Logger } = require('@hmcts/nodejs-logging');
```

Comment:
Could this `{ }` be moved to the top of the file for consistency

Reply:
Done